tests/integration/test_core_integration.py

```diff
@@ -19,6 +20,50 @@ def _set_up_and_tear_down(peewee_db):
     clean_db()
 
 
+def test_get_image_predictions():
+    logo_annotation1 = LogoAnnotationFactory(image_prediction__image__barcode="123")
+    logo_annotation2 = LogoAnnotationFactory(image_prediction__image__barcode="456")
+    logo_annotation = LogoAnnotationFactory(type="label")
```

Comment:
As above if we want to be able to test all filters, we do want to use ImagePredictionFactory to have ImagePrediction instances without LogoAnnotation attached to them.

There again type that we care about is ImagePrediciton type, not LogoAnnotation type.